<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakEditDialog</class>
 <widget class="QDialog" name="BreakEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>545</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Commissions</string>
  </property>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>700</width>
     <height>124</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;table border=0 width=&quot;100%&quot;&gt;
&lt;tr&gt;&lt;th colspan=2 align=&quot;center&quot;&gt;Variables&lt;/th&gt;&lt;th&gt;&lt;/th&gt;&lt;th colspan=2 align=&quot;center&quot;&gt;Operators&lt;/th&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;H =&gt; Head&lt;/td&gt;&lt;td&gt;T =&gt; Total&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;td&gt;+ =&gt; add&lt;/td&gt;&lt;td&gt;- =&gt; subtract&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;P =&gt; Price&lt;/td&gt;&lt;td&gt;W =&gt; Weight&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;td&gt;* =&gt; multiply&lt;/td&gt;&lt;td&gt;/ =&gt; divide&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>420</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Commisson Applies to:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>500</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </widget>
  <widget class="Q3TextEdit" name="commissionDescripText">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>221</y>
     <width>310</width>
     <height>270</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Q3Table" name="breakTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>370</width>
     <height>270</height>
    </rect>
   </property>
   <property name="numRows">
    <number>11</number>
   </property>
   <property name="numCols">
    <number>5</number>
   </property>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Expression</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Break Start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>-</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Min</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Max</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/commissiondao.h</include>
  <include location="local">dao/animalcodedao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
